components/org.wso2.carbon.identity.oauth.uma.permission.endpoint/src/main/java/org/wso2/carbon/identity/oauth/uma/permission/endpoint/PermissionApiServiceImpl.java

```diff
@@ -127,11 +140,11 @@ private boolean isValidTokenScope(MessageContext context) {
     }
 
     /**
-     * Retrieve resource owner name
+     * Retrieve resource owner name with user domain
```

Comment:
Add a full stop at the end

Reply:
fixed: https://github.com/wso2-extensions/identity-oauth-uma/pull/68